src/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <div class="top line"></div>
+  <div class="middle line"></div>
+  <div class="bottom line"></div>
```

Comment:
Main classes should be written first.
```suggestion
  <div class="block bottom"></div>
```